fastlane/lib/fastlane/helper/git_helper.rb

```diff
@@ -139,6 +139,17 @@ def self.git_branch_name_using_HEAD
       end
     end
 
+    # Returns the default git remote branch name
+    def self.git_default_remote_branch_name
```

Comment:
Can we add an option on here to choose the remote it looks it? 😊 Some of my repos have multiple remotes

And then also add this as an option on the action?

Reply:
Sure, I can look into that.

Reply:
Done & includes a unit test and example code.